mak/Makefile.gather

```diff
@@ -353,12 +354,12 @@ gather-openldap-master: pre-manifest
 
 
 gather-expat-release: pre-manifest
-	expat_srcpath=https://github.com/libexpat/libexpat/releases && \
-	expat_ver=`wget -q -O - $$expat_srcpath | sed -n 's#^.*<a.* href=".*\/archive\/refs\/tags\/R_\(.*\)\.tar\.gz".*$$#\1#;t once;$$!b;:once;p;q;'` && \
-	expat_srcpath=https://github.com/libexpat/libexpat/archive/refs/tags/R_$$expat_ver.tar.gz && \
-	expat_srcdir=libexpat-R_$$expat_ver && \
-	expat_pkg=$$expat_srcdir.tar.gz && \
-	expat_ver=`echo $$expat_ver | sed "s#_#.#g"` && \
+	gitpath=https://github.com/libexpat/libexpat && \
+	gitver=`wget --max-redirect=0 $$gitpath/releases/latest/ 2>&1 | sed 's#Location: .*/tag/\([^/]*\) \[following\]#\1#p;d;'` && \
+	expat_ver=`echo $$gitver | sed 's#^R_##;s#_#.#g;'` && \
+	expat_srcdir=expat-$$expat_ver && \
+	expat_pkg=$$expat_srcdir.tar.bz2 && \
```

Comment:
This was the purpose of the tar -xzf -> -xjf change in Makefile.download - and because the
distribution package is in bzip2 format and it gains the results of the autotools distribution
pass plus an associated .asc gpg signature, vs. the github source ball tar-up we used to use.